content/en-us/tutorials/fundamentals/coding-5/intro-to-arrays.md

```diff
@@ -120,7 +120,7 @@ Use index values to assign specific pieces of dialogue to the NPC. To use a valu
    ```
 
    <Alert severity="warning">
-   If an index doesn't exist, a nil error will display in Output. For example, since this array has three values, using index[4] will result in an error.
+   If an index doesn't exist, a nil error will display in Output. For example, since this array only has three values, using dialogueArray[4] will result in an error.
```

Comment:
```suggestion
   If an index doesn't exist, a nil error will display in Output. For example, since this array only has three values, using `dialogueArray[4]` will result in an error.
```